src/VirtoCommerce.CustomerExportImportModule.Web/Scripts/blades/import-preview.js

```diff
@@ -77,7 +75,6 @@ angular.module('virtoCommerce.customerExportImportModule')
                     };
                     dialogService.showDialog(dialog, "Modules/$(VirtoCommerce.CustomerExportImport)/Scripts/dialogs/customerImport-dialog.tpl.html", "platformWebApp.confirmDialogController");
                 },
-                permission: blade.importPermission
```

Comment:
Why you removed permission instead of replacing it?

Reply:
Becaue this is a preview blade. I implement permission on previous stage, on the main customer blade for export and import buttons.